lib/credo/check/readability/no_list_sigils.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Credo.Check.Readability.NoListSigils do
+  @moduledoc """
+  A check to suggest explicit lists over the ~w and ~W sigls.
+  """
+  use Credo.Check,
+    param_defaults: [],
+    explanations: [
+      check: ~S"""
+      Although the ~w and ~W sigils allow for brevity when writing code, code is
+      read many more times than it is written. List definitions are both relatively
+      brief and completely unambiguous; consider defining lists explicitly rather than
+      via the sigil.
```

Comment:
```suggestion
      Code is read more times than it is written and although the ~w and ~W sigils 
      allow for brevity in code, explicit list definitions have the benefit that they are 
      both relatively brief and completely unambiguous.
```

I think that leading with the benefits of what NOT to do weakens the reasoning behind this check. Instead we should put the reading > writing argument first :+1:

Reply:
cd9c39f7